docs/examples/openebs.md

```diff
@@ -0,0 +1,181 @@
+# OpenEBS
+
+This tutorial covers the installation of OpenEBS as a Helm extension, both from
+scratch and how to migrate it from a storage extension.
+
+## Installing OpenEBS from scratch
+
+**WARNING**: Do not configure OpenEBS as both a storage extension and Helm
+extension. It's is considered an invalid configuration k0s will entirely ignore
+the configuration to prevent accidental upgrades or downgrades. The chart
+objects defined in the API will still behave normally.
+
+OpenEBS can be installed as a helm chart by adding it as an extension to your configuration:
+
+```yaml
+  extensions:
+    helm:
+      repositories:
+      - name: openebs-internal
+        url: https://openebs.github.io/charts
+      charts:
+      - name: openebs
+        chartname: openebs-internal/openebs
+        version: "3.9.0"
+        namespace: openebs
+        order: 1
+        values: |
+          localprovisioner:
+            hostpathClass:
+              enabled: true
+              isDefaultClass: false
+```
+
+If you want OpenEBS to be your default storage class set `isDefaultClass` to `true`.
+
+## Migrating bundled OpenEBS to helm extension
+
+The bundled OpenEBS extension is already a helm extension installed as a
+`chart.helm.k0sproject.io`, for this reason all we need to do is removing the
+manifests and cleaning the object. However this must be done in a specific order
+to prevent data loss.
+
+**WARNING**: Not following the steps in the precise order presented by the
+documentation may provoke data loss.
+
+The first step to perform the migration is disabling the applier-manager component
+in every controller. For every controller, stop the controller and start it again
+with the flag `--disable-components=applier-manager`. If you already had this flag
+set it to `--disable-components=<previous value>,applier-manager`.
+
+Once the applier-manager is disabled in every running controller, you need to modify
+the configuration to use `external_storage` instead of `openebs_local_storage`.
```

Comment:
```suggestion
Once the `applier-manager` is disabled on all controllers, you need to modify
the k0s configuration to use `external_storage` instead of
`openebs_local_storage`.
```